### DISCUSSION TITLE
Support MIT OpenCourseWare

### DISCUSSION THREAD

**Discussion by LordJohn42** [Nitrino/easysubs#220] [Ideas]
https://ocw.mit.edu/courses/18-06-linear-algebra-spring-2010/resources/lecture-1-the-geometry-of-linear-equations/
As example of course. Try to realize this functionality for plugin, but dont promise finish this.

**Comment by Nitrino** [comment]
Hi @LordJohn42 
I think it's a great resource and definitely worth adding support for!
If you decide to add the service yourself, pay attention, now in the master version 2.0, this is a completely new version and still raw, better yet use a branch with [the version v1.x](https://github.com/Nitrino/easysubs/tree/v1.0) 

And feel free to ask any questions about implementation!
